spec/models/form_profile_spec.rb

```diff
@@ -13,6 +13,9 @@
   before do
     stub_evss_pciu(user)
     described_class.instance_variable_set(:@mappings, nil)
+    Flipper.disable(:va_v3_contact_information_service)
+    Flipper.disable(:remove_pciu)
+    Flipper.disable('remove_pciu_2')
     Flipper.disable(:disability_526_toxic_exposure)
     Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_PPIU_DIRECT_DEPOSIT)
```

Comment:
Flipper will be tested in the form_profile_v2_spec in this https://github.com/department-of-veterans-affairs/vets-api/pull/18729